src/components/seeds/database/EditColumns.tsx

```diff
@@ -191,7 +191,6 @@ const sections: Section[] = [
     name: 'Withdrawal',
     options: [
       [COLUMNS_INDEXED.withdrawalDate, COLUMNS_INDEXED.withdrawalQuantity],
-      [COLUMNS_INDEXED.withdrawalDestination, COLUMNS_INDEXED.remainingQuantity],
```

Comment:
we could move withdrawalPurpose here, so we have the three columns

Reply:
Great suggestion, you got it